### ISSUE TITLE
[Feature] How to make it to have vertical stepper

### ISSUE THREAD

**Issue by AbhaysinghBhosale** [newbreedofgeek/react-stepzilla#164]
**Expected behavior**
Need to have option where we can pass prop and convert it as vertical stepper.

**Additional context**
Trying to use it as vertical stepper but not able to convert is with css as both stepper and components are in single wrapper. Is there any way to have prop by which we can convert stepper to vertical one

**Comment by newbreedofgeek** [comment]
@AbhaysinghBhosale i like the idea of vertical stepper, have you not been able to achieve this with CSS?

**Comment by AbhaysinghBhosale** [comment]
@newbreedofgeek Yes. I did made it vertical but not with only css. As just with css its not possible cos for vertical we need 2 rows
 I did overwritten this library file and converted it into vertical stepper with some additional css.

Will be converting it to change based on props. so with just prop should be able to make it vertical and horizontal
